main_app/tests/test_app.py

```diff
@@ -0,0 +1,13 @@
+from django.test import TestCase
+import requests
+
+
+class AnimalTestCase(TestCase):
+
+    def test_can_make_api_call(self):
+        """Can make an API call and expect specific attributes"""
+        data = requests.get('https://jsonplaceholder.typicode.com/todos/1').json()
+        # import code; code.interact(local=dict(globals(), **locals()))
+        self.assertEqual(str(type(data)), "<class 'dict'>")
```

Comment:
What does class 'dict' stand for or what does it mean?

Reply:
@judithpillado It's what Python calls a "hash"

Reply:
oh okay thanks for explaining! Awesome work